tests/test_lru_cache.py

```diff
@@ -0,0 +1,157 @@
+from pathlib import Path
+
+import pytest
+
+from mopidy_tidal.lru_cache import LruCache, SearchCache
+
+
+@pytest.fixture
+def lru_cache(config):
+    cache_dir = config["core"]["cache_dir"]
+    return LruCache(max_size=8, persist=True, directory="cache")
+
+
+def test_props(config):
+    l = LruCache(max_size=1678, persist=True, directory="cache")
```

Comment:
Out of curiosity, how come we have set "max_size=1678" here and 1679 further down?

Reply:
I was just testing that the `max_size` parameter is actually passed through to `LruCache().max_size`.  The numbers were random.